myshopmatic.com.website.json

```diff
@@ -1,21 +1,21 @@
 {  
-   "providerId":"goshopmatic.hk",
+   "providerId":"goshopmatic.com",
    "providerName":"Shopmatic",
    "serviceId":"website",
    "serviceName":"Shopmatic Site",
    "logoUrl":"https://domainconnect.org/wp-content/uploads/2016/07/shopmatic-300x81.png",
    "description":"Enables a domain to work with Shopmatic",
-   "variableDescription":"v1 is domain name being configured to work with shopmatic. v2 is the shopmatic hosted name",
+   "variableDescription":"INPUTDOMAIN is domain name being configured to work with shopmatic.",
+   "syncBlock":false,
+   "shared":true,
+   "warnPhishing":true,
+   "hostRequired":false,
    "records":[  
-      {  
-         "type":"REDIR301",
-         "target":"www.%v1%"
-      },
       {  
          "type":"CNAME",
          "host":"www",
-         "pointsTo":"%v2%",
+         "pointsTo":"%INPUTDOMAIN%",
          "ttl":"3600"
       }
    ]
-}
+}
```

Comment:
Where is the apex A record?  Don't they need this too?

Reply:
I asked the provider about this.  They insisted they don't want one in the template for now.  They have 3 templates in GoDaddy's system that aren't in gitHub, and none of them have A records.